Nodejs/Product/Npm/SPI/NodeModules.cs

```diff
@@ -167,6 +148,31 @@ internal class NodeModules : AbstractNodeModules {
 
             return depth;
         }
+
+        private static IEnumerable<KeyValuePair<string, IPackageJson>> GetTopLevelDependencies(string modulesBase) {
```

Comment:
Rename to `GetTopLevelPackageDirectories` - these aren't really the top level dependencies in npm v3.